src/components/Copyright/Copyright.js

```diff
@@ -35,7 +35,8 @@ function Copyright({ layerService, format, ...other }) {
         layerService
           .getLayersAsFlatArray()
           .filter(l => l.getVisible())
-          .map(l => l.getCopyright()),
+          .map(l => l.getCopyright())
+          .filter(cr => cr !== undefined),
```

Comment:
```suggestion
          .filter(cr => cr),
```
The copyright can be `"",` e.g. when it comes from Mapbox source and `sourceCache.used` is `false` see https://github.com/geops/react-spatial/pull/418

Reply:
So you want to filer on '' and false as well?

Reply:
@edellucien Yes, I want to filter out `""`. And in which case is it `false`?